web_scraper.py

```diff
@@ -1,65 +1,72 @@
-            gender = indent.find('strong').text
-        elif (indent.text.startswith("Waga")):
-            weight = indent.find('strong').text
-        elif (indent.text.startswith("Numer")):
-            number = indent.find('strong').text
-        elif (indent.text.startswith("Status")):
-            status = indent.find('strong').text
-        elif (indent.text.startswith("Przyjęty")):
-            date_in = indent.find('strong').text
-        elif (indent.text.startswith("Wydany")):
-            date_out = indent.find('strong').text
-        elif (indent.text.startswith("Znaleziony")):
-            address_found = indent.find('strong').text
-        elif (indent.text.startswith("Boks")):
-            box = indent.find('strong').text
-        elif (indent.text.startswith("Grupa")):
-            group_name = indent.find('strong').text
-            group_link = indent.find('a').get('href')
-    #to be removed but may be useful in debug:
-#     print(f'''
-# Breed: {breed}
-# Age: {age}
-# Gender: {gender}
-# Weight: {weight}
-# Number: {number}
-# Status: {status}
-# date_in: {date_in}
-# date_out: {date_out}
-# address_found: {address_found}
-# box: {box}
-# group_name: {group_name}
-# group_link: {group_link}
-# ''')
+        if indent.text.startswith("W typie rasy"):
+            info["breed"] = indent.find('strong').text if indent.find('strong') else None
+        elif indent.text.startswith("Wiek"):
+            info["age"] = indent.find('strong').text if indent.find('strong') else None
+        elif indent.text.startswith("Płeć"):
+            info["gender"] = indent.find('strong').text if indent.find('strong') else None
+        elif indent.text.startswith("Waga"):
+            info["weight"] = indent.find('strong').text if indent.find('strong') else None
+        elif indent.text.startswith("Numer"):
+            info["number"] = indent.find('strong').text if indent.find('strong') else None
+        elif indent.text.startswith("Status"):
+            info["status"] = indent.find('strong').text if indent.find('strong') else None
+        elif indent.text.startswith("Przyjęty"):
+            info["date_in"] = indent.find('strong').text if indent.find('strong') else None
+        elif indent.text.startswith("Wydany"):
+            info["date_out"] = indent.find('strong').text if indent.find('strong') else None
+        elif indent.text.startswith("Znaleziony"):
+            info["address_found"] = indent.find('strong').text if indent.find('strong') else None
+        elif indent.text.startswith("Boks"):
+            info["box"] = indent.find('strong').text if indent.find('strong') else None
+        elif indent.text.startswith("Grupa"):
+            info["group_name"] = indent.find('strong').text if indent.find('strong') else None
+            info["group_link"] = indent.find('a').get('href') if indent.find('a') else None
+    return info
+
 
+# debugging print function
+def print_pet_info(info):

```

Comment:
Not necessarily needed. Just printing the dictionary will show all keys and values.